_summaries/hedman2021baking_2.md

```diff
@@ -13,7 +13,38 @@ TODO: Summarize the paper:
 * How well does the paper perform?
 * What interesting variants are explored?
 
+## Core Idea
+View synthesis is the task of given a set of images of a specific scene, generating a 3d representation of that scene. An innovative model in this task is NeRF, however this model is slow and intensive. This paper presents SNeRG, a model thats two orders of magnitude faster than NeRF. This is done by trading computation for storage through pre-training.
```

Comment:
I think SNeRG is a data structure rather than a model. If I am wrong, please correct me

Reply:
Right!